Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2018-05-29 -- v3.0.1
+- Add support for lobal data collection switch in FIRApp.
```

Comment:
It looks like this won't be included in Core's release this time around, so we can save this for next time.

Reply:
The change has pushed to master tho, would that be an issue if we skip the release this time?

https://github.com/firebase/firebase-ios-sdk/pull/1219

Reply:
It's been pushed to master but it isn't in the public header, so developers shouldn't be using it. Functionality wise, Messaging stays the same since that flag will always return `YES` if nothing is set.

Although the bits are there, it's essentially a no-op until Core moves the API to the public header.

Reply:
Done.